.github/workflows/test-package.yml

```diff
@@ -362,7 +371,7 @@ jobs:
           diff -qr _c src/
           diff -qr _dart lib/third_party
       - name: Generate full bindings
-        run: dart run jnigen --config jnigen_full.yaml
+        run: dart run jnigen --config jnigen.yaml --override classes="org.apache.pdfbox.pdmodel;org.apache.pdfbox.text"
```

Comment:
Why is this needed?

Reply:
I am committing only 2-3 required classes to reduce code.

But in CI we have freedom to generate bindings for the entire `pdmodel` and `text` subpackages and ensure that renaming etc.. are scaling well.